tools/blender/2.9x/docs/guide.md

```diff
@@ -0,0 +1,199 @@
+
+![](./021.png)
+
+The blderik Level3 is now again the child object of the armature, since the hammer bone is a child of the armature.
+
+![](./022.png)
+
+Now repeat the whole bone duplication, alignment, and parenting for the piston object blderik Level2. You should end up with something like this:
+
+![](./023.png)
+
+We will not make a bone for the blderik Level 1, as we do not need to animate the base structure and it will remain static. Now we are prepared for animating.
+
+Switch to Pose mode, switch to view the derik from the side, hold down shift and select both Piston and hammer bone. Now make sure the frame 0 is selected in the timeline, and right-click in the view and select Insert Keyframe.
+
+![](./024.png)
+
+Select Location, Rotation & Scale. When you select the bones again, you can see the keyframe data on both of them.
+
+Now let's select the frame 30, rotate the hammer bone, move the piston up on the z-axis, and insert another keyframe.
+
+![](./025.png)
+
+Now go to frame 60, rotate the hammer back and move the piston back into the same position as on frame 0. If you want to be sure, you can copy the frame 0 by right-clicking the points and selected keyframe and Copy. Then right-click frame 60 and paste.
+
+![](./026.png)
+![](./027.png)
+
+Now set the end frame to 59. 
+
+![](./028.png)
+
+Why 59? Because frame 60 is the same location and rotation as frame 0, and we want the animation to be circular. Now you can press space to play the animation and ESC to stop it, or use the play stop buttons.
+
+![](./029.png)
+
+The animation is done, and we can do the export. Switch back to object mode. We need to set the cycles and frame time for both The Level3 “hammer” and the Level2 “piston” objects. Animation Cycles sets the number of repetitions for the animation. In our case, we leave it on 0. That means infinite repetitions. Animation Time is actually the time for one frame in milliseconds. Let's say we want the animation cycle to take two seconds (2000 milliseconds) to play. And we have 60 frames. 2000 / 60 = 33.33 So we can set the value to 33. Set the same value for both Level3 “hammer” and the Level2 “piston.” That way they will be in sync.
+
+![](./030.png)
+
+Now let's export the PIE. Switch to object mode, select the armature. Change Pie Version to 3 and make sure events are cleared.
+
+![](./031.png)
+
+With the selected root armature, click on File – Export – Warzone 2100 model (.pie), select the location and click export pie to directory.
+
+![](./032.png)
+
+### Postprocessing
+
+Now we have to do some cleanup because the export is causing two small issues. It adds empty lines to the file and adds additional unnecessary frame to the animations. Open the exported blderik_anim.pie in your preferred text editor and remove empty lines – usually on end of level polygons or Animobject frames.
```

Comment:
```suggestion
Now we have to do some cleanup because the export is causing two small issues. It adds empty lines to the file and adds additional unnecessary frame to the animations. Open the exported `blderik_anim.pie` in your preferred text editor and remove empty lines – usually on end of level polygons or Animobject frames.
```